### DISCUSSION TITLE
How do I delete a user in Flutter?

### DISCUSSION THREAD

**Discussion by Tomic-Riedel** [supabase/supabase#3358] [Questions] [answered]
Hey.
I just have the problem that I make a button so that the user can delete his account. Unfortunately with "await supabase.auth[......]" I can only logout a user (await supabase.auth.signOut() ) 
Unfortunately it doesn't work like in JavaScript that you can write "supabase.auth.api.deleteUser()" or "supabase.auth.deleteUser()" to delete a user. 

So my question is: How can I delete a user with Supabase in Flutter?

Edit: I am very sorry that I did not ask this question in the supabase-flutter repository, it was an oversight 😬

**Comment by silentworks** [comment, marked as answer]
So this function has been discussed for the `gotrue-dart` library which in turn would make it available to `supabase-flutter`, but It seems there was a question surrounding its use case and no move had been made on it since then.

https://github.com/supabase/gotrue-dart/pull/26
